eng/pipelines/templates/steps/fetch-package-list.yml

```diff
@@ -0,0 +1,28 @@
+parameters: 
```

Comment:
Declare `WorkingDirectory` here and have it default to `$(System.DefaultWorkingDirectory)` so it doesn't have to be specified in invocations of the template.

Reply:
The working location is the docs repo instead azure sdk repo. So we use `${{ parameters.DocRepoLocation }}` instead of `$(System.DefaultWorkingDirectory)`.

Reply:
Don't want to add it right now. Prefer to add necessary ones only, as newly added parameters do not introduce breaking changes.

Reply:
Chatted offline... Parameters make it easier to understand the template interface :)